src/azure-cli/azure/cli/command_modules/serviceconnector/_utils.py

```diff
@@ -5,7 +5,7 @@
 
 import time
 from knack.log import get_logger
-from knack.util import todict
+from knack.util import todict, CLIError
```

Comment:
CLIError is deprecated.

Reply:
We don't throw CLIError but just catch the existing one.